docs/config.md

```diff
@@ -235,6 +235,7 @@ tlsSkipVerify  |     N     |  bool       | false   | Whether to skip TLS server
 tlsClientCert  |     N     |  string     | ""      | Client cert for client authentication
 tlsClientKey   |     N     |  string     | ""      | Client key for client authentication
 saslEnabled    |     N     |  bool       | false   | Whether to enable SASL
+saslMechanism  |     N     |  string     | ""      | if empty or not used the SASL Mechanism is disabled, if specified the SASL Mechanism can be SCRAM-SHA-256 or SCRAM-SHA-512
```

Comment:
> if empty or not used the SASL Mechanism is disabled

this seems misleading. If empty, we default to the "PLAIN" mechanism
How about this:

```suggestion
saslMechanism  |     N     |  string     | ""      | if unset, the PLAIN mechanism is used. You can also specify SCRAM-SHA-256 or SCRAM-SHA-512.
```

we could also allow to explicitly set 'PLAIN' but i don't feel strongly about this.